package.json

```diff
@@ -3,6 +3,6 @@
     "version": "0.1.0",
     "author": "Playbook",
     "devDependencies": {
-        "percy": "0.28.4"
+        "percy": "5.0.0"
```

Comment:
5 major version bump~

Reply:
Lockfile got shrinked massively tho~

Reply:
Apparently, that because of [this reason](https://www.npmjs.com/package/percy).
It seems that the package got renamed to `@percy/cli`.

Reply:
I switched to `@percy/cli-upload` ([pkg](https://www.npmjs.com/package/@percy/cli-upload?activeTab=dependencies)) to minimize deps, can we give it a try? Dont look like we need config update

Reply:
Sure, that sounds good.

Reply:
Error still wants percy 5.0.. hmmmhmhmh what did I miss

Reply:
Ah, cli is yappari requirement, fixed, please rerun 🙇🏻

Reply:
Finally succeeded! There you go